book/tutorials/nsidc-data-access.md

```diff
@@ -0,0 +1,3 @@
+#NSIDC DATA ACCESS Tutorial
+
+#Content Here
```

Comment:
```suggestion
# NSIDC DATA ACCESS Tutorial

## Content Here
```
Jupyterbook is picky about titles (see https://guidebook.hackweek.io/resources/tutorial-resources.html#formatting). You'll also want to hit the 'resolve conflicts' button on this PR to get the most recent table of contents